modules/microshift-install-rpms.adoc

```diff
@@ -20,30 +20,24 @@ Use the following procedure to install {microshift-short} from an RPM package.
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos \
-    --enable rhocp-<4.18>-for-<9>-$(uname -m)-rpms \ # <1>
-    --enable fast-datapath-for-<9>-$(uname -m)-rpms # <2>
+    --enable rhocp-{ocp-version}-for-rhel-{op-system-version-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-rhel-${op-system-version-major}-$(uname -m)-rpms
 ----
-<1> Replace _<4.18>_ and _<9>_ with the compatible versions of your {microshift-short} and {op-system-base-full}.
-<2> Replace  _<9>_ with the compatible version of {op-system-base}.
 
 . For extended support (EUS) releases, also enable the EUS repositories by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-`$ sudo subscription-manager repos \
-    --enable rhel-<9>-for-x86_64-appstream-eus-rpms \ # <1>
-    --enable rhel-<9>-for-x86_64-baseos-eus-rpms` # <2>
+$ sudo subscription-manager repos \
+    --enable rhel-{op-system-version-major}-for-x86_64-appstream-eus-rpms \
+    --enable rhel-{op-system-version-major}-for-x86_64-baseos-eus-rpms`
 ----
-<1> Replace _<9>_ with the compatible major version number of {op-system-base}.
-<2> Replace _<9>_ with the compatible major version number of {op-system-base}.
 
 . Avoid unintended future updates into an unsupported configuration by locking your operating system version with the following command:
 +
-[source,terminal]
-----
-$ sudo subscription-manager release --set=<9.4> command. # <1>
+[source,terminal,subs="+quotes"]
 ----
-<1> Replace _<9.4>_ with the major and minor version of your compatible {op-system-base} system.
+$ sudo subscription-manager release --set={op-system-version} command.
```

Comment:
Looks like this is not rendered properly.

Reply:
I'm asking in our team channel about this. The attribute is correct, seeing if someone else has insight. I suspect we need to escape the = somehow, but I'm not certain we really can. Checking.

Reply:
Resolved, noticed that extra word in the command and removed. Should be good once it rebuilds.